src/vscode-dts/vscode.proposed.remoteCodingAgents.d.ts

```diff
@@ -3,6 +3,27 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// empty placeholder for coding agent contribution point from core
+declare module 'vscode' {
+	export interface RemoteCodingAgentInformation {
+		id: number;
+		number: number;
+		title: string;
+		user: {
+			login: string;
+		};
+		html_url: string;
+		state: string;
+		created_at: string;
+		updated_at: string;
+	}
 
-// @joshspicer
+	export interface RemoteCodingAgentInformationProvider extends Disposable {
```

Comment:
Take a look at the chat provider proposal too (`LanguageModelChatProvider2` mainly I think) for general direction